README.md

```diff
@@ -129,6 +129,10 @@ When a feed is being used hypercore will emit `interested` with the correspondin
 
 This is emitted when a feed is no longer being used.
 
+#### `core.use(extension)`
+
+Use an extension to the protocol. See section below.
```

Comment:
I think it would be helpful to add an overview of the API, if we think that it is stable.